task.yaml

```diff
@@ -4,12 +4,10 @@ task:
   # docker image settings, required section
   container:
     # image name to start on miner, required param
-    name: image:v1
+    name: cocaine/cocaine-core:latest
```

Comment:
can you return the sample data back, please

Reply:
Ok, falling back to a non-working config.